composer.json

```diff
@@ -10,32 +10,24 @@
       "email": "opensource@10up.com"
     }
   ],
-  "repositories": [
-    {
-        "type":"package",
-        "package": {
-          "name": "georgestephanis/application-passwords",
-          "version":"0.1.1",
-          "source": {
-              "url": "https://github.com/WordPress/application-passwords.git",
-              "type": "git",
-              "reference":"aaa502d70190d13fbc45d58ff47db1635f7dbe1c"
-            }
-        }
-    }
-  ],
   "require": {
     "php": ">=5.6",
     "yahnis-elsts/plugin-update-checker": "^4.4",
-    "georgestephanis/application-passwords": "0.1.1",
-    "ext-json": "*"
+    "ext-json": "*",
```

Comment:
@dinhtungdu Is there a reason the `composer.json` and `composer.lock` files have changes as part of this PR? Doesn't seem like they are needed, so I'm assuming these were just accidentally brought in.

Reply:
It's my fix to bundle Application password properly. Currently, it locks to 0.1.1. But I agree that it isn't relevant to this PR.